patent.py

```diff
@@ -0,0 +1,40 @@
+import argparse
+import requests
+import json
+
+
+def patent(**kwargs):
```

Comment:
The function name should indicate what the function **does**, not what it is. I would recommend you create a class called `Patents`, and have this function inside (rename the function to `get` or something like that)

Also, I cheated and read ahead a little bit, and you likely don't need to capture **kwargs here. Try to limit your input to what you'll actually use. See a future comment for more info here.